Tools/llm-tool/LLMTool.swift

```diff
@@ -203,29 +206,88 @@ struct EvaluateCommand: AsyncParsableCommand {
 
     @MainActor
     mutating func run() async throws {
-        let (modelContainer, modelConfiguration) = try await memory.start(args.load)
+        let modelContainer = try await memory.start { [args] in
+            try await args.load(
+                defaultModel: "mlx-community/Mistral-7B-v0.1-hf-4bit-mlx",
```

Comment:
We should update this default model, it's pretty dated. Maybe to `mlx-community/Mistral-7B-Instruct-v0.3-4bit` is a good option?

Reply:
Sure, I will give it a run and make sure it works!

- [ ] test this

Reply:
It is one of the preset models, so good to go